theano/sandbox/softsign.py

```diff
@@ -1,35 +0,0 @@
-from __future__ import absolute_import, print_function, division
```

Comment:
To help port old code, can you add a new file with the same name that will import softsign to allow old code to still work?